app/uk/gov/hmrc/gform/eval/EvaluationResults.scala

```diff
@@ -51,13 +52,16 @@ case class EvaluationResults(
     expr: FormCtx,
     recData: RecData[SourceOrigin.OutOfDate],
     fromVariadicValue: VariadicValue => ExpressionResult
-  ): ExpressionResult =
-    exprMap
-      .get(expr)
-      .getOrElse(
-        recData.variadicFormData
-          .get(expr.formComponentId.modelComponentId)
-          .fold(ExpressionResult.empty)(fromVariadicValue))
+  ): ExpressionResult = {
+    val result = recData.variadicFormData
```

Comment:
This result may not be needed, why to compute it all the time?

Reply:
Accidental commit. changed for debugging but forgot to remove.

Reply:
removed